lib/workers/pr/index.js

```diff
@@ -253,6 +253,8 @@ async function ensurePr(prConfig) {
         const platformOptions = {
           azureAutoComplete: config.azureAutoComplete,
           statusCheckVerify: config.statusCheckVerify,
+          mergeWhenPipelineSucceeds:
+            config.automerge && config.automergeType === 'pr',
```

Comment:
Shouldn't this come from the config?

Reply:
With the current approach setting the `allowMergeWhenPipelineSucceeds` option is independent of the existing `automerge` option. To get something automerged, enable `automerge` , to gain a bit speed, additionallyy enable MWPS.

Reply:
But you’re not checking for config.allowMerge..... here. Or is it because it’s set at initRepo time and doesn’t need to be checked here also?

Reply:
Yes, it’s currently handled within the platform because it saw it more as a policy that only the platform should care about. But now that’s it’ll become a regular config option I can also add this check here and remove it from the platform.

Reply:
Done with ea783946d0d76f953c6fe3b4c3f21026d2318010